src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentMethodInfoPlugin.java

```diff
@@ -16,12 +16,12 @@
 
 package org.killbill.billing.plugin.adyen.api;
 
-import java.util.UUID;
```

Comment:
>  there should be no change if the code was properly formatted (which is sadly not the case here).

I just reformatted the file in my IDE and it didn't trigger any change (the [import ordering](https://github.com/killbill/killbill/blob/master/.idea/codeStyleSettings.xml#L14-L29) looks good, no?).

> It's no fun to contribute if you always have to fear that pressing "format all" will kill every diff :(

But it's no fun either to review a PR with 23 changed files, 1,595 additions and 798 deletions, especially for a bug fix that tricky. I'm having a hard time understanding functionally what changed.